ios/Rakefile

```diff
@@ -1,5 +1,5 @@
 FULL_TESTS = ENV['FULL_TESTS'] != 'false'
-BUILD_SDK = ENV['BUILD_SDK'] || 'iphonesimulator13.4'
+BUILD_SDK = ENV['BUILD_SDK'] || 'iphonesimulator13.5'
```

Comment:
CI was failing so I had to do this change.

```xcodebuild: error: SDK "iphonesimulator13.4" cannot be located.```

Reply:
This is that thing that we discussed previously in the squad where they make a change and sims are no longer available on GH Actions so this is all good by me!